### DISCUSSION TITLE
Lora rx tx Lora or Lora rx tx Ht

### DISCUSSION THREAD

**Discussion by ghost** [lora-aprs/LoRa_APRS_iGate#147] [Q&A]
Hi. does this lora igate only relate to Lora to lora only. like where I all use AX.25 from different types of radios. Such as argentdata anytone and international space station in the form of AX.25. Digi Path usually uses WIDE1-1, WIDE2-2.

**Comment by peterus** [comment]
Hi, this project is just about LoRa! No 2m or something else.
It is using the APRS ASCII format which is used on the aprs servers (no AX.25)
